Controller/Payment/FormDataAnalytics.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace Razorpay\Magento\Controller\Payment;
+
+use Razorpay\Magento\Model\Config;
+use Magento\Framework\Controller\ResultFactory;
+use Razorpay\Magento\Model\TrackPluginInstrumentation;
+use Razorpay\Magento\Model\PaymentMethod;
+
+
+/**
+ * Using TrackPluginInstrumentation and sending events to Segment
+ *
+ * ...
+ */
+class FormDataAnalytics extends \Razorpay\Magento\Controller\BaseController
+{
+    protected $setup;
+
+    protected PaymentMethod $paymentMethod;
+
+    protected TrackPluginInstrumentation $trackPluginInstrumentation;
+
+    /**
+     * @param \Magento\Framework\App\Action\Context $context
+     * @param \Magento\Customer\Model\Session $customerSession
+     * @param \Magento\Checkout\Model\Session $checkoutSession
+     * @param \Razorpay\Magento\Model\Config $config
+     * @param \Psr\Log\LoggerInterface $logger
+     * @param Razorpay\Magento\Model\TrackPluginInstrumentation
+     */
+    public function __construct(
+        \Magento\Framework\App\Action\Context $context,
+        TrackPluginInstrumentation $trackPluginInstrumentation,
```

Comment:
@nikhilsharma1100 php error occurred with message class `TrackPluginInstrumentation` not found.